crates/worker/src/partition_processor_manager.rs

```diff
@@ -390,11 +390,13 @@ impl<T: TransportConnect> PartitionProcessorManager<T> {
         networking: Networking<T>,
         bifrost: Bifrost,
     ) -> Self {
+        let queue_length = updateable_config.pinned().worker.internal_queue_length();
```

Comment:
I'm afraid such a large value (1000 by default) is too dangerous to use. Under load, external components (namely CC) will not be interested in the response of this long-enqueued messages anymore but PPM will still have to go through them, this _could_ render us in a busy loop doing useless work.

Reply:
True. We probably have this problem at all places where we have a somewhat high-capacity buffer. I guess the TCP send buffer has a similar effect.

The main idea here was to mitigate a bit the negative effect that an unresponsive `PartitionProcessorManager` has on other message handlers by blocking the network reactor if its capacity is only 2. The best solution would probably be to make the `PPM` less blocking.